schemas/event.yml

```diff
@@ -704,3 +704,19 @@
         Alert events, indicated by `event.kind:alert`, are a common use case for this field.
 
       example: https://mysystem.mydomain.com/alert/5271dedb-f5b0-4218-87f0-4ac4870a38fe
+
+    - name: reason
+      level: extended
+      type: keyword
+      short: The reason captured by the event
```

Comment:
Perhaps we should phrase this more naturally. E.g. 

"Reason why this event happened, according to the source"